torch/_refs/__init__.py

```diff
@@ -1138,6 +1138,34 @@ def clamp(
     raise ValueError(msg)
 
 
+@out_wrapper
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self", "min"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def clamp_min(
+    self: TensorLikeType,
+    min: Optional[TensorOrNumberLikeType] = None,
+) -> TensorLikeType:
+    self, min = _maybe_broadcast(self, min)
+
+    return maximum(self, min)
```

Comment:
to preserve gradient behavior (clamp doesn't spread gradients when boundary and input are the same) it's better to use `where`

Reply:
Good catch! So we should also change `clamp` to `where` as well.

> clamp doesn't spread gradients when boundary and input are the same

I think clamp does propagate gradient when input/bound equals. But the grad behavior is indeed different from minimum/maximum

```
In [31]: x = torch.ones(2, 2).requires_grad_()

In [32]: torch.clamp_max(x, torch.tensor(1.0)).sum().backward()

In [33]: x.grad
Out[33]:
tensor([[1., 1.],
        [1., 1.]])

In [34]: x = torch.ones(2, 2).requires_grad_()

In [35]: torch.minimum(x, torch.tensor(1.0)).sum().backward()

In [36]: x.grad
Out[36]:
tensor([[0.5000, 0.5000],
        [0.5000, 0.5000]])
```